Developer-Guide.md

```diff
@@ -569,6 +570,52 @@ file exists and contains details of the image and how it was created:
 $ cat /var/lib/osbuilder/osbuilder.yaml
 ```
 
+## Capturing kernel boot logs
+
+Sometimes it is useful to capture the kernel boot messages from a Kata Container
+launch. If the container launches to the point whereby you can `exec` into it, and
+if the container has the necessary components installed, often you can use `dmesg` at
+a shell prompt inside the container to view the kernel boot logs.
+
+If however you are unable to `exec` into the container, you can enable some debug
+options to have the kernel boot messages logged into the system journal.
+
+To achieve this you need to `enable_debug=true` in the Kata Configuration file
+for both the hypervisor/kernel and the proxy configuration sections. The following
+shows the differences as required in the `configuration-qemu.toml` file for instance:
+
+```patch
```

Comment:
oii, I wouldn't like to include a path here, because configuration files tends to change, hence the patch context may not match causing hunks . Keeping this patch up-to-date can be tedious and painful.
There is already a section in this file that mentions how to enable debugging mode [1], how about we mention it instead of this patch?

[1] - https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md#enable-full-debug

Reply:
ack. Reworked to show the two different sets of debug that might need to be enabled.